.buildkite/release.steps.yaml

```diff
@@ -0,0 +1,102 @@
+---
+common: &common
+  agents:
+    - "capable_of_building=gdk-for-unreal"
+    - "environment=production"
+    - "permission_set=builder"
+    - "platform=linux"  # if you need a different platform, configure this: macos|linux|windows.
+    - "queue=${CI_LINUX_BUILDER_QUEUE:-v4-2019-12-12-bk5225-daecba805768d787}"
+    - "scaler_version=2"
+    - "working_hours_time_zone=london"
+
+  timeout_in_minutes: 20
+  retry:
+    automatic:
+        # This is designed to trap and retry failures because agent lost connection. Agent exits with -1 in this case.
+      - exit_status: -1
+        limit: 3
+
+steps:
+  # TODO: I've added engine-version & preview keys. I need to call these using "buildkite-agent meta-data get" where required.
+  - block: Configure your release.
+    prompt: Fill out these details for your release.
+    fields:
+      - text: UnrealGDK component release version
+        key: gdk-version
+        required: true
+        hint: The UnrealGDK component release version you want to create a release candidate for, e.g. - 0.1.2 or 1.0.1.
+        
+      - select: "Unreal Engine version(s)"
+        key: "engine-version"
+        hint: "The Unreal Engine versions you want to create a release candidate for, e.g. - 4.24-SpatialOSUnrealGDK"
+        required: true
+        multiple: true
+        default:
+          - "4.24-SpatialOSUnrealGDK"
```

Comment:
According to Buildkite's [block step](https://buildkite.com/docs/pipelines/block-step#select-input-attributes) documentation, yes you can:

> `default` | The value of the option or **options** that will be pre-selected. When `multiple` is enabled, this can be an **array** of values to select by default.

Reply:
> This PR is already too big as it is... Could you please break it into different functional or value areas?

@michelefabris & I spoke offline. We agree that this will be an ordeal to peer review, but we can't see a way to ship a smaller unit of functionality. I'm already not going to include "automatically populate CHANGELOG.md", this will be a follow up.

I will pair with reviewers and run them through how this works.